### PR TITLE
Grammar/font/rewording tweaks to the first part of "Git Internals" ch…

### PR DESCRIPTION
**Pull request by rpjday**
…apter.

Tweaks include:

 - monospaced font for file, directory and command names
 - hyphenation
 - minor grammar rewording
 - "objects database" -> "object database"
 - adding some first-term rendering in a few places
 - update contents of freshly-created .git directory

### CODE REVIEW

File: book/10-git-internals/sections/plumbing-porcelain.asc [hunk 1]
```diff
@@ -17,21 +17,22 @@ Here's what it looks like:
...
-HEAD
-config*
+branches/
+config
 description
+HEAD
 hooks/
 info/
 objects/
 refs/
```
Comment (ben) [inline book/10-git-internals/sections/plumbing-porcelain.asc]: Interesting. Where are you seeing a `branches` folder? Here's the output from my machine:

```
$ git init foo
Initialized empty Git repository in /private/tmp/foo/.git/
$ cd foo/.git
$ ls -F1
HEAD
config
description
hooks/
info/
objects/
refs/
$ git --version
git version 2.14.1
```

File: book/10-git-internals/sections/plumbing-porcelain.asc [hunk 2]
```diff
@@ -17,21 +17,22 @@ Here's what it looks like:
...
 
 You may see some other files in there, but this is a fresh `git init` repository – it's what you see by default.
-The `description` file is only used by the GitWeb program, so don't worry about it.
-The `config` file contains your project-specific configuration options, and the `info` directory keeps a global exclude file (((excludes))) for ignored patterns that you don't want to track in a .gitignore file.
+The `description` file is used only by the GitWeb program, so don't worry about it.
+The `config` file contains your project-specific configuration options, and the `info` directory keeps a global exclude file (((excludes))) for ignored patterns that you don't want to track in a `.gitignore` file.
 The `hooks` directory contains your client- or server-side hook scripts, which are discussed in detail in <<_git_hooks>>.
 
-This leaves four important entries: the `HEAD` and (yet to be created) `index` files, and the `objects` and `refs` directories.
+This leaves four important entries: the `HEAD` and (yet to be created) `index` file, and the `objects` and `refs` directories.
```
Comment (ben) [inline book/10-git-internals/sections/plumbing-porcelain.asc]: Well, "files" here is referring to "`HEAD` and `index`", so it still makes sense for it to be plural.

File: book/10-git-internals/sections/plumbing-porcelain.asc (hunk 1 above)
Comment (rpjday) [reply book/10-git-internals/sections/plumbing-porcelain.asc]: I just created a new repo on my Fedora 26 box (git --version = 2.13.6), and I get a branches/ folder even in a new repo. Are you saying you don't?

File: book/10-git-internals/sections/plumbing-porcelain.asc (hunk 2 above)
Comment (rpjday) [reply book/10-git-internals/sections/plumbing-porcelain.asc]: Ah, quite so, will revert.

**Pull request merged**